ops/lightly_run.py

```diff
@@ -0,0 +1,107 @@
+import json
+from dagster import op
+from lightly.api import ApiWorkflowClient
+from lightly.openapi_generated.swagger_client.models.dataset_type import DatasetType
+from lightly.openapi_generated.swagger_client.models.datasource_purpose import DatasourcePurpose
+
+
+with open("config/config.json", "r") as jsonfile:
+    configs_json = json.load(jsonfile)
+    
+LIGHTLYTOKEN = configs_json["LIGHTLYTOKEN"]
+DATASETNAME = configs_json["DATASETNAME"]
+S3REGION = configs_json["S3REGION"]
+S3ROLEARN = configs_json["S3ROLEARN"]
+S3SEXTERNALID = configs_json["S3SEXTERNALID"]
+S3INPUTBUCKET = configs_json["S3INPUTBUCKET"]
+S3LIGHTLYBUCKET = configs_json["S3LIGHTLYBUCKET"]
+
+class LightlyRun:
+    """Pexels client to download a random popular video. """
+
+    def __init__(
+        self,
+        lightly_token = LIGHTLYTOKEN,
+        dataset_name = DATASETNAME,
+        s3_region = S3REGION,
+        s3_role_arn = S3ROLEARN,
+        s3_external_id = S3SEXTERNALID,
+        s3_input_bucket = S3INPUTBUCKET,
+        s3_lightly_bucket = S3LIGHTLYBUCKET
+
+    ):
+        self.lightly_token = lightly_token
+        self.dataset_name = dataset_name
+        self.s3_region = s3_region
+        self.s3_role_arn = s3_role_arn
+        self.s3_external_id = s3_external_id
+        self.s3_input_bucket = s3_input_bucket
+        self.s3_lightly_bucket = s3_lightly_bucket
+        self.client = ApiWorkflowClient(token=lightly_token)
+
+    def createDataset(self) -> None:
```

Comment:
Python uses snake case for functions, see https://google.github.io/styleguide/pyguide.html#316-naming